qiskit_experiments/library/calibration/fine_drag.py

```diff
@@ -0,0 +1,265 @@
+
+    # section: see_also
+        qiskit_experiments.library.calibration.drag.DragCal
+
+    # section: reference
+        .. ref_arxiv:: 1 1612.00858
+        .. ref_arxiv:: 2 1011.1949
+    """
+
+    __analysis_class__ = FineDragAnalysis
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        r"""Default values for the fine amplitude experiment.
+
+        Experiment Options:
+            repetitions (List[int]): A list of the number of times that Rp - Rm gate sequence
+                is repeated.
+            schedule (ScheduleBlock): The schedule for the plus rotation.
+            normalization (bool): If set to True the DataProcessor will normalized the
+                measured signal to the interval [0, 1]. Defaults to True.
+        """
+        options = super()._default_experiment_options()
+        options.repetitions = list(range(20))
+        options.schedule = None
+
+        return options
+
+    @classmethod
+    def _default_analysis_options(cls) -> Options:
+        """Default analysis options."""
+        options = super()._default_analysis_options()
+        options.normalization = True
+        options.angle_per_gate = 0.0
+        options.phase_offset = -np.pi / 2
+        options.amp = 1.0
+
+        return options
+
+    def __init__(self, qubit: int):
+        """Setup a fine amplitude experiment on the given qubit.
+
+        Args:
+            qubit: The qubit on which to run the fine amplitude calibration experiment.
+        """
+        super().__init__([qubit])
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply before repeating the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+    @staticmethod
+    def _post_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply after repeating the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+    def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
+        """Create the circuits for the fine DRAG calibration experiment.
+
+        Args:
+            backend: A backend object.
+
+        Returns:
+            A list of circuits with a variable number of gates. Each gate has the same
+            pulse schedule.
+        """
+        schedule, circuits = self.experiment_options.schedule, []
+
+        drag_gate = Gate(name=schedule.name, num_qubits=1, params=[])
+
+        for repetition in self.experiment_options.repetitions:
+            circuit = self._pre_circuit()
+
+            for _ in range(repetition):
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+
+            circuit.ry(np.pi / 2, 0)  # Maps unwanted Z rotations to qubit population.
+
+            circuit.compose(self._post_circuit(), inplace=True)
```

Comment:
What is a case where post_circuit would be used? None of the subclasses here use it.

Reply:
I moved the ry rotation to `_post_circuit` here 1da11ed . The rational behind this move is that it will make it easier to implement a fine DRAG cal on the EF subspace when we wish to implement such an experiment. Like this the EF experiment will only need to override the pre and post circuit methods.